### PR TITLE
Filings/Reports table document download link

### PR DESCRIPTION
**Pull request by xtine**
Download link for CSV and PDF documents on Filings and Reports datatables. #1610

screen shot 2016-10-17 at 1 24 38 pm
screen shot 2016-10-17 at 1 24 51 pm

### CODE REVIEW

**Comment by jenniferthibault** [comment]
One style/pattern request:

Could we use the exact same pattern we enlisted for the mini calendar button? The difference I'm seeing now is just that the download button does not have the dividing line between the icon and the arrow, so would need that added :

screen shot 2016-10-17 at 4 47 12 pm

**Comment by jenniferthibault** [comment]
Yikes that was a big screenshot of a tiny button. Not sure how that happened!

**Comment by xtine** [comment]
@jenniferthibault: updated button with the dividing line:

screen shot 2016-10-17 at 2 40 53 pm

**Comment by xtine** [comment]
Functional dropdowns: screen shot 2016-10-17 at 10 36 27 pm

**Comment by xtine** [comment]
Mobile ready dropdowns: 
screen shot 2016-10-17 at 10 23 46 pm

File: static/js/modules/columns.js [hunk 1]
```diff
@@ -428,11 +428,22 @@ var reports = {
     className: 'all column--xl',
```
Comment (xtine) [inline static/js/modules/columns.js]: Updated.

File: static/js/modules/columns.js [hunk 2]
```diff
@@ -255,20 +257,18 @@ var filings = {
   },
   pdf_url: {
```
Comment (xtine) [inline static/js/modules/columns.js]: I had this in mind but didn't get to it. I renamed to `document_type` to be generically descriptive.

**Comment by xtine** [comment]
@noahmanger: this is ready for the most part, but still concerned about the financial columns clashing into each other for some of the report pages (mainly Presidential and PAC and party committee) at ~1200px wide.

**Pull request merged**